src/static/css/cf-theme-overrides.less

```diff
@@ -35,7 +35,7 @@
 @link-underline:                @pacific;
 @link-text-visited:             @teal;
 @link-underline-visited:        @teal;
-@link-text-hover:               @pacific-50;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5864db12a15d93104442e3b6c26bcd388f5a0e09 cc @duelj in case the category slug links should be darker than other links on hover for some reason.